packages/pytest-simcore/src/pytest_simcore/helpers/utils_pylint.py

```diff
@@ -0,0 +1,15 @@
+""" Helps with running Pylint tests on different modules """
+import subprocess
+
+
+def assert_pylint_is_passing(pylintrc, package_dir, autodetect=0):
```

Comment:
MINOR: ``autodect=0`` is gives the impression to be a flag to enable/disable "autodectection" which is not the case. It denotes the number of parallel jobs in pylint and it happens that "0" indicates auto-detection.  I would do something like

```python
AUTODETECT = 0

def assert_pylint_is_passing(pylintrc, package_dir, *, number_of_jobs: int = AUTODECT):
   ...

```

Reply:
why add the `*`  to it?

Reply:
what do you mean? I like having options as enforced key-arguments